packer_files/vsphere_centos_template/build.pkr.hcl

```diff
@@ -0,0 +1,13 @@
+packer {
+  required_plugins {
+    vsphere = {
+      source  = "github.com/hashicorp/vsphere"
+      version = ">= 0.0.1"
```

Comment:
Change to a fixed version

Reply:
please see the recommendation here:
https://www.packer.io/docs/builders/vsphere

Reply:
I don't think this is a recommendation, I think it's an example.